inc/tools.php

```diff
@@ -385,19 +386,19 @@ function cptui_get_post_type_code( $cptui_post_types = [], $single = false ) {
 			$key      = key( $cptui_post_types );
 			$callback = 'cptui_register_my_cpts_' . str_replace( '-', '_', esc_html( $cptui_post_types[ $key ]['name'] ) );
 		}
-?>
+		?>
 
 function <?php echo esc_html( $callback ); ?>() {
-<?php
-// Space before this line reflects in textarea.
+		<?php
+		// Space before this line reflects in textarea.
 		foreach ( $cptui_post_types as $type ) {
-			echo cptui_get_single_post_type_registery( $type );
+			echo cptui_get_single_post_type_registery( $type ); // phpcs:ignore.
 		}
-?>
+		?>
 }
 
 add_action( 'init', '<?php echo esc_html( $callback ); ?>' );
-<?php
+		<?php
```

Comment:
Can you check on this one with line 389-401, given the comment I put regarding space in textareas? Basically spacing provided outside of php tags gets output to the page.